src/swing.ts

```diff
@@ -80,9 +80,10 @@ export interface Stack {
    * Creates an instance of Card and associates it with an element.
    *
    * @param {HTMLElement} element
+   * @param {boolean} prepend
    * @return {Card}
    */
-  createCard(elment: HTMLElement): void;
+  createCard(elment: HTMLElement, prepend: boolean): void;
```

Comment:
thanks @rajeevriitm thats what I was thinking as well. @Sparkomatic can we update it likewise?